src/OpenSage.Viewer/UI/Views/CsfView.cs

```diff
@@ -38,6 +48,32 @@ public override void Draw(ref bool isGameViewFocused)
             ImGui.Columns(1, null, false);
         }
 
+        private void UpdateSearch(string searchText)
```

Comment:
Add a Search to the `CsfViewer`, because it was impossible/ extremly difficult to find the hardcoded labels otherwise.